### PR TITLE
ALS-6330: Fix variant explorer functionality

### PR DESCRIPTION
**Pull request by ramari16**
Variant explorer functionality was broken as part of the initial genomic rewrite. This addresses those issues. Additionally, this adds integration tests from the 1k genome data set (ALS-4643)

### CODE REVIEW

**Review by Luke-Sikina — COMMENTED** [review]
I can't build this

```
 /home/luke/code/bch/pic-sure-hpds/common/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/crypto/CryptoDefaultKeyTest.java:[19,10] cannot find symbol
[ERROR]   symbol:   class BeforeTestClass
[ERROR]   location: class edu.harvard.hms.dbmi.avillach.hpds.crypto.CryptoDefaultKeyTest

```
What am I doing wrong?

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/AbstractProcessor.java [hunk 1]
```diff
@@ -336,6 +342,7 @@ public PhenoCube<?> load(String key) throws Exception {
 										return ret;
 									}else {
 										log.warn("ColumnMeta not found for : [{}]", key);
+										log.info(Joiner.on(",").join(phenotypeMetaStore.getColumnNames()));
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/AbstractProcessor.java:345]: Delete

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/GenomicProcessor.java [hunk 2]
```diff
@@ -31,4 +28,7 @@ public interface GenomicProcessor {
     Set<String> getInfoStoreValues(String conceptPath);
 
     List<InfoColumnMeta> getInfoColumnMeta();
+
+    // todo: make the map value a Set instead of array
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/GenomicProcessor.java:32]: Delete comment

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/VariantIndexCache.java [hunk 3]
```diff
@@ -80,7 +80,7 @@ public VariantIndex load(String infoColumn_valueKey) throws IOException {
                 int x = 0;
                 for(Integer variantIndex : variantIndexIntArray) {
                     // todo: shouldn't this be greater than or equal to 0? 0 is a valid index
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/VariantIndexCache.java:82]: delete comment

File: processing/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/processing/VariantUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+class VariantUtilsTest {
+
+    @Test
+    public void pathIsVariantSpec_oldFormat() {
+        assertTrue(VariantUtils.pathIsVariantSpec("chr21,5032061,A,G"));
+    }
+    @Test
+    public void pathIsVariantSpec_newFormat() {
+        assertTrue(VariantUtils.pathIsVariantSpec("chr21,5032061,A,G,LOC102723996,missense_variant"));
+    }
```
Comment (ramari16) [inline processing/src/test/java/edu/harvard/hms/dbmi/avillach/hpds/processing/VariantUtilsTest.java:16]: Add not matching tests

File: service/src/main/resources/application-bdc-auth.properties [hunk 5]
```diff
@@ -1,6 +1,7 @@
 SMALL_JOB_LIMIT = 100
 SMALL_TASK_THREADS = 1
 LARGE_TASK_THREADS = 1
+VCF_EXCERPT_ENABLED=true
```
Comment (ramari16) [inline service/src/main/resources/application-bdc-auth.properties:4]: Rollback until we build new data set

**Pull request merged**